requirements.txt

```diff
@@ -0,0 +1,15 @@
+pytest
+pandas
+biopython==1.79
+deepspeed==0.5.9
+dm-tree==0.1.6
+ml-collections==0.1.0
+numpy==1.21.2
+PyYAML==5.4.1
+requests==2.26.0
+scipy==1.7.1
+tqdm==4.62.2
+typing-extensions==3.10.0.2
+pytorch_lightning==1.5.10
+wandb==0.12.21
+DLLogger@git+https://github.com/NVIDIA/dllogger#egg=dllogger
```

Comment:
we can't have this one in requirements.txt because that will break the pip build (pypi doesn't accept this kind of stuff)